e2e/optimism_test.go

```diff
@@ -269,20 +270,36 @@ func TestOptimismGenericCommitmentMigration(t *testing.T) {
 	proxyTS, shutDown := testutils.CreateTestSuite(tsConfig)
 	defer shutDown()
 
+	expectedWriteCount := uint64(0)
+	expectedReadCount := uint64(0)
+
 	ot := actions.NewDefaultTesting(t)
 
 	optimism := NewL2AltDA(ot, proxyTS.Address(), true)
 	exerciseGenericCommitments(t, ot, optimism)
-	requireDispersalRetrievalEigenDA(
-		t,
-		proxyTS.Metrics.HTTPServerRequestsTotal,
-		commitments.OptimismGeneric)
+	expectedWriteCount++
+	expectedReadCount++
+	actualWriteCount, err := proxyTS.Metrics.HTTPServerRequestsTotal.Get(
+		string(commitments.OptimismGeneric), http.MethodPost)
+	require.NoError(t, err)
+	require.Equal(t, expectedWriteCount, actualWriteCount)
+	actualReadCount, err := proxyTS.Metrics.HTTPServerRequestsTotal.Get(
+		string(commitments.OptimismGeneric), http.MethodGet)
+	require.NoError(t, err)
+	require.Equal(t, expectedReadCount, actualReadCount)
```

Comment:
nit: I think it would still be nice for readability of tests to put this behind a function like
`requireReadWriteCounts(writeCount, readCount int)`

Reply:
I'm not sure the complexity of the checks warrants a method with the many params, but I don't feel strongly here. 

[a9c836b](https://github.com/Layr-Labs/eigenda-proxy/pull/348/commits/a9c836bfb5f5a2b750439e8eddcf9760fbfa793a)

Reply:
oh yeah I see... heh